### PR TITLE
Timer functionality as suggested a month ago

### PR DESCRIPTION
**Pull request by FweeGamerHD**
hi, it's me, the guy who suggested the dynamically changing weather.

I added a command to run a timer where it rolls a 75% chance for some weather to occur (rain 66%, thunder 33%) and then every 1-2min there's a 5% chance for the weather to worsen (only applies if raining) and 15% of the weather getting better (irrelevant for clear weather)

I have tested it and it *should* work, however I'm not 100% sure about the rolls every few minutes when a weather is active.

### CODE REVIEW

**Review by Lucaslah — CHANGES_REQUESTED** [review]

**Comment by Lucaslah** [comment]
Thanks for the PR, I will do some testing and let you know

File: common/src/main/java/me/lucaslah/weatherchanger/commands/WeatherChangerCommand.java [hunk 1]
```diff
@@ -48,6 +48,15 @@ public <T> void register(CommandDispatcher<T> dispatcher) {
                 })
         );
 
+        command.then(LiteralArgumentBuilder.<T>literal("toggleTimer")
+                .executes(context -> {
+                    WeatherChanger.toggleTimer();
+                    String message = WeatherChanger.isTimerEnabled() ? "Timer enabled." : "Timer disabled.";
```
[the remark this answers is not in the document]
Comment (FweeGamerHD) [reply common/src/main/java/me/lucaslah/weatherchanger/commands/WeatherChangerCommand.java:54]: Definitely! I can do that no problem.

[the remark this answers is not in the document]
Comment (Lucaslah) [reply common/src/main/java/me/lucaslah/weatherchanger/commands/WeatherChangerCommand.java:54]: Thanks

**Review by Lucaslah — APPROVED** [review]
Looks good

**Comment by Lucaslah** [comment]
Thanks for your contribution

**Pull request merged**